addons/mail/static/src/models/messaging_notification_handler.js

```diff
@@ -24,6 +24,20 @@ registerModel({
          * the current users. This includes pinned channels for instance.
          */
         start() {
+            this.mailboxInfos = [
+                {
+                    mailbox: this.messaging.inbox,
+                    hasMessage: message => message.isNeedaction,
+                },
+                {
+                    mailbox: this.messaging.history,
```

Comment:
I don't think history even has a counter, do we actually care?

Reply:
It is not display indeed and as proposed I have hard coded the update of the 2 displayed counters (starred and inbox).